build.rs

```diff
@@ -0,0 +1,14 @@
+use std::{env, fs, path::PathBuf};
+
+fn main() {
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+    println!("cargo:rustc-link-search)={}", out_dir.display());
+
+    fs::copy("ld/stub.x", out_dir.join("stub.x")).unwrap();
+    println!("cargo:rerun-if-changed=ld/stub.x");
+    println!("cargo:rustc-link-arg=-Tld/stub.x");
+
+    fs::copy("ld/rom.x", out_dir.join("rom.x")).unwrap();
```

Comment:
So I would rework this to be `rom_esp32c3.x` or something like that, but I guess you'll already be doing something like that if you're adding esp32 support?